plugins/mcreator-localization/lang/texts.properties

```diff
@@ -2192,6 +2194,15 @@ elementgui.command.type=Type:
 elementgui.command.permission_level=<html>Execution permission level:<br><small>Minimal level for the command to run
 elementgui.command.warning.empty_string=Command can''t be empty string
 elementgui.command.arguments=Command parameters and actions builder
+elementgui.damagetype.damage_properties=Damage properties
+elementgui.damagetype.exhaustion=<html>Exhaustion amount:<br><small>Default value is 0.1
+elementgui.damagetype.scaling=Does damage scale with difficulty?
+elementgui.damagetype.effects=Damage sound effects:
```

Comment:
I would use singular form in UI since despite this having plural form in code/JSON, effectively it defines single "damage sound effect" and does not accept list of them

Reply:
Fair, I went with the plural because it can play different variations of the sound